### PR TITLE
[rom_ext_e2e] Sketch bootstrap test plan in rom_ext/e2e/README.md

### PR DESCRIPTION
**Pull request by dmcardle**

### CODE REVIEW

**Comment by dmcardle** [comment]
Hey @alphan, I figured the next step for ROM_EXT bootstrap is to get started on e2e tests. I sketched out an informal test plan. PTAL!

File: sw/device/silicon_creator/rom_ext/e2e/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    | # | Enabled in OTP | Bootable silicon owner image | Expect bootstrap |
+    |---|----------------|------------------------------|------------------|
+    | 1 | No             | None                         | No               |
+    | 2 | No             | Slot A                       | No               |
+    | 3 | No             | Slot B                       | No               |
+    | 3 | No             | Slot A and B                 | No               |
+    | 4 | Yes            | None                         | Yes              |
+    | 5 | Yes            | Slot A                       | No               |
+    | 6 | Yes            | Slot B                       | No               |
+    | 3 | Yes            | Slot A and B                 | No               |
```
Comment (alphan) [inline sw/device/silicon_creator/rom_ext/e2e/README.md:20]: nit: ~3~ -> 6

File: sw/device/silicon_creator/rom_ext/e2e/README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+* ROM_EXT bootstrap will not erase the chip until it receives a `CHIP_ERASE` or
+  `SECTOR_ERASE` SPI command.
+
+* ROM_EXT bootstrap will neither erase nor program the ROM_EXT region of slot A.
+
+* ROM_EXT bootstrap will erase and program the ROM_EXT region of slot A.
```
Comment (alphan) [inline sw/device/silicon_creator/rom_ext/e2e/README.md:27]: ~ROM_EXT~ -> owner/BL0

File: sw/device/silicon_creator/rom_ext/e2e/README.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+* ROM_EXT bootstrap will neither erase nor program the ROM_EXT region of slot B.
+
+* ROM_EXT bootstrap will erase, but will not program the ROM_EXT region of slot A.
```
Comment (alphan) [inline sw/device/silicon_creator/rom_ext/e2e/README.md:31]: ~ROM_EXT~ -> owner/BL0
~slot A~ -> slot B

**Comment by dmcardle** [comment]
Resolved a few silly mistakes. This PR is just a Markdown file, so I'll merge it without CI checks.

**Review by alphan — APPROVED** [review]

**Pull request merged**